hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1797,6 +1797,40 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.s3g.secret.http.enabled</name>
+    <value>false</value>
+    <tag>OZONE, S3GATEWAY</tag>
+    <description>The boolean which enables the Ozone S3Gateway Secret endpoint.
+    </description>
+  </property>
+  <property>
+    <name>ozone.s3g.secret.http.auth.type</name>
+    <value>simple</value>
```

Comment:
I think we should set this default to "kerberos", not "simple", for simple is not allowed in the implementation.